modules/core/04-channel/types/expected_keepers.go

```diff
@@ -92,6 +94,9 @@ type ConnectionKeeper interface {
 		connectionHops []string,
 		keyGenerator KeyGenFunc,
 	) error
+
+	GetCounterpartyConnectionHops(exported.ConnectionI, *commitmenttypes.MsgMultihopProofs) ([]string, error)
```

Comment:
Not sure if it is okay to add these extra interface methods? Maybe the implementations could be moved elsewhere if so?